bigbluebutton-html5/imports/ui/components/settings/submenus/closed-captions/service.js

```diff
@@ -0,0 +1,29 @@
+import Storage from '/imports/ui/services/storage/session';
+import Captions from '/imports/api/captions';
+
+getClosedCaptionSettings = () => {
+  let ccSettings = {};
+  let ccEnabled = Storage.getItem('closedCaptions');
+  if(ccEnabled ==  true) {
```

Comment:
If you want to ensure that the value is a bool you can just double not it.

`ccSettings.ccEnabled = !!ccEnabled`